### ISSUE TITLE
Let Debug Console go to side like split editors

### ISSUE THREAD

**Issue by sdegutis** [microsoft/vscode#118975]
Issue Type: <b>Feature Request</b>

I often want to see more vertical than horizontal output when debugging stuff. Being able to put the debug console on the right side of the editor, instead of under it, would help me a lot.

VS Code version: Code 1.54.2 (fd6f3bce6709b121a895d042d343d71f317d74e7, 2021-03-11T00:56:09.783Z)
OS version: Darwin x64 19.6.0

**Comment by IllusionMH** [comment]
You can drag'n'drop debug console to the sidebar (as other views): either to action bar or inside of Run and Debug view.

Does it resolve your request?

**Comment by sdegutis** [comment]
If you mean the sidebar with the Explorer, Search, Extensions (etc) icons/views, no, that's too small, and also I'd like to keep that open. I'm not sure what you mean by Run and Debug view though.

**Comment by IllusionMH** [comment]
Run & Debug view in sidebar
![image](https://user-images.githubusercontent.com/1312662/111164909-257e5100-85a7-11eb-84bf-f74ead4896bc.png)

Can be dragged right after variables or any other part of debug view

**Comment by sdegutis** [comment]
Yeah that would still be too small of an area. Plus I'd like to keep the other action bar views open/available.

**Comment by gjsjohnmurray** [comment]
@sdegutis it sounds like you want https://github.com/microsoft/vscode/issues/26777#issuecomment-768948599

**Comment by sdegutis** [comment]
Yep, exactly. In fact I was surprised that isn't already possible.
